fiftyone/zoo/datasets/base.py

```diff
@@ -1173,6 +1174,206 @@ def _get_raw_dir(self, dataset_dir):
         return os.path.join(root_dir, "raw")
 
 
+class SamaCOCODataset(FiftyOneDataset):
+    """Sama-COCO is a large-scale object detection, segmentation, and captioning
+    dataset based on COCO2017. It is a relabeling of the original training and
+    validation sets with tighter polygons and more individually annotated crowds.
+
+    This version contains images, bounding boxes and segmentations
+    for Sama's version of the 2017 version of the dataset.
+
+    This dataset supports partial downloads:
+
+    -   You can specify subsets of data to download via the ``label_types``,
+        ``classes``, and ``max_samples`` parameters
+    -   You can specify specific images to load via the ``image_ids`` parameter
+
+    See :ref:`this page <dataset-zoo-sama-coco>` for more information about
+    partial downloads of this dataset.
+
+    Full split stats:
+
+    -   Train split: 118,287 images
+    -   Test split: 40,670 images
+    -   Validation split: 5,000 images
+
+    Notes:
+
+    -   COCO defines 91 classes but the data only uses 80 classes
+    -   Some images from the train and validation sets don't have annotations
+    -   The test set does not have annotations
+    -   Sama-COCO may have some discrepancies with COCO-2017 in terms of the instances labeled
+
+    #TODO: update example usage
```

Comment:
Update example to use sama-coco instead

Reply:
All refs to original coco have been resolved (dataset source, naming, examples)